flang/mod_with_common.mod

```diff
@@ -0,0 +1,6 @@
+﻿!mod$ v1 sum:9b05914eee59feea
```

Comment:
Should we be adding "precompiled" .mod files like this?

Also is this the correct and intended directory?

Reply:
Nope. This should not have been added here. Git add goof.

Reply:
File removed.